api/src/libs/migrateBusinessesFromUsers.ts

```diff
@@ -0,0 +1,45 @@
+import { BusinessesDataClient, UserDataClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { processBusinessesMigration } from "@libs/migrationUtils";
+import { UserData } from "@shared/userData";
+
+export const migrateBusinessesFromUsers = async (
+  userDataClient: UserDataClient,
+  businessesDataClient: BusinessesDataClient,
+  logger: LogWriterType
+): Promise<void> => {
+  const processBatch = async (users: UserData[]): Promise<void> => {
+    for (const userData of users) {
+      logger.LogInfo(`Starting migration for userId: ${userData.user.id}`);
```

Comment:
Boulder, there was a comment around, tracking when a migration for a user or business is complete. This way, if the fn crashes, we could rerun it and it would skip the user or business that is already copied over.